data/SAMPLE-cat.html

```diff
@@ -1,208 +1,149 @@
-                     Alternative name
-                     </b></td>
-               <td width="70%">Bicycle</td>
+               <th>Alternative name</th>
+               <td>Bicycle</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Origin
-                     </b></td>
-               <td width="70%"></td>
+               <th>Origin</th>
+               <td></td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Definition
-                     </b></td>
-               <td width="70%">A human-powered, pedal-driven, single-track vehicle, having two wheels attached to
-                  a frame, one behind the other.
-               </td>
+               <th>Definition</th>
+               <td>A human-powered, pedal-driven, single-track vehicle, having two wheels attached to a frame, one behind the other.</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Origin of definition
-                     </b></td>
-               <td width="70%"></td>
+               <th>Origin of definition</th>
+               <td>&nbsp;</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Date
-                     </b></td>
-               <td width="70%"></td>
+               <th>Date</th>
+               <td>&nbsp;</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Identifier
-                     </b></td>
-               <td width="70%"></td>
+               <th>Identifier</th>
+               <td>&nbsp;</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Population
-                     </b></td>
-               <td width="70%"></td>
+               <th>Population</th>
+               <td>&nbsp;</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Indication quality
-                     </b></td>
-               <td width="70%"></td>
+               <th>Indication quality</th>
+               <td>&nbsp;</td>
             </tr>
             <tr>
-               <td width="30%"><b>
-                     Abstract object
-                     </b></td>
-               <td width="70%">No</td>
+               <th>Abstract object</th>
+               <td>No</td>
             </tr>
          </tbody>
       </table>
       
-      <div class="subheader">AttributeType overview</div>
+      <h3>AttributeType overview</h3>
       
       <table>
+         <thead>
+            <colgroup>
+               <col width="20%"></col>
+               <col width="50%"></col>
+               <col width="15%"></col>
+               <col width="15%"></col>
+            </colgroup>
+            <tr>
+               <th>Attribute name</th>
+               <th>Definition</th>
+               <th>Format/type</th>
+               <th>Mult</th>
```

Comment:
`<th>Mult</th>` in plaats van `<td><i>Mult</i></th>`. Als de heading toch anders moet zjin qua layout, dan kan dit worden opgelost via een `class =""` in de `<table>` en in de css via:
```css
table.classname th {
    colour: #000000;
    font-style: italic;
}
```